iroh-net/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-//! Peer-to-peer connectivity based on QUIC.
+//! Peer-to-peer connectivity QUIC connections.
```

Comment:
```suggestion
//! Peer-to-peer QUIC connections.
```

Not sure about my suggestion.
But the current state basically reads like two subjects back-to-back in a sentence.